source/includes/stackpath/_instances.md

```diff
@@ -0,0 +1,117 @@
+      "id": "e485fe8c-e1de-40f7-b3b3-a43700441e0e/instance-name-1",
+      "status": "RUNNING"
+    }
+  ],
+  "metadata": {
+    "recordCount": 1
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/instance?workloadId=:workloadId</code>
+
+Retrieve a list of all workload instance in a given [environment](#administration-environments) for a given workload id.
+
+Query Params | &nbsp;
+---- | -----------
+`workloadId`<br/>*string* | The workload ID to get the instances for. It is mandatory.
+
+Attributes | &nbsp;
+------- | -----------
+`id`<br/>*string* | The instance id. It is the combination of the instance workloadId and instance name.
+`name`<br/>*string* | The instance name.
+`status`<br/>*string* | The instance status. Possible values are: RUNNING, STARTING, SCHEDULING.
+`stackId`<br/>*UUID* | The stack ID to which belongs the instance.
+`workloadId`<br/>*UUID* | The workload ID to which belongs the instance.
+`ipAddress`<br/>*string* | The internal IP address associated to the instance.
+`publicIpAddress`<br/>*string* | The public IP address associated to the instance.
+`location`<br/>*string* | The instance location.
+`createdDate`<br/>*string* | The instance creation date.
+`startedDate`<br/>*string* | The instance started date.
+
+
+<!-------------------- RETRIEVE AN INSTANCE -------------------->
+
+#### Retrieve an instance
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/stackpath/test-area/instances/e485fe8c-e1de-40f7-b3b3-a43700441e0e/instance-name-1"
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": {
+    "stackId": "asd04459-1ccf-4a0d-8aaa-98d0f52dd2f3",
+    "workloadId": "e485fe8c-e1de-40f7-b3b3-a43700441e0e",
+    "name": "instance-name-1",
+    "ipAddress": "10.128.16.2",
+    "publicIpAddress": "151.139.71.25",
+    "location": "Los Angeles",
+    "createdDate": "2020-11-10T16:52:15Z",
+    "startedDate": "2020-11-10T16:52:15Z",
+    "id": "e485fe8c-e1de-40f7-b3b3-a43700441e0e/instance-name-1",
+    "status": "RUNNING"
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/instances/:id</code>
+
+Retrieve an instance in a given [environment](#administration-environments) with the specific id.
+
+Attributes | &nbsp;
+------- | -----------
+`id`<br/>*string* | The instance id. It is the combination of the instance workloadId and instance name.
+`name`<br/>*string* | The instance name.
+`status`<br/>*string* | The instance status. Possible values are: RUNNING, STARTING, SCHEDULING.
+`stackId`<br/>*UUID* | The stack ID to which belongs the instance.
+`workloadId`<br/>*UUID* | The workload ID to which belongs the instance.
+`ipAddress`<br/>*string* | The internal IP address associated to the instance.
```

Comment:
Same as above to --> with

Reply:
e27dc4c55ea93c46d14d801c5736954b2954be64